stdlib/public/SwiftShims/MetadataSections.h

```diff
@@ -0,0 +1,55 @@
+//===--- RuntimeShims.h - Access to runtime facilities for the core -------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+
+#ifndef SWIFT_STDLIB_SHIMS_METADATASECTIONS_H
+#define SWIFT_STDLIB_SHIMS_METADATASECTIONS_H
+
+#include "SwiftStddef.h"
+#include "SwiftStdint.h"
+
+#ifdef __cplusplus
+namespace swift { extern "C" {
+#endif
+
+struct MetadataSectionRange {
```

Comment:
Why break this out of struct MetadataSections? It might make for a smaller diff to just keep the definition nested.